jekyll/_cci2/config-translator.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: classic-docs
+title: "Using the 1.0 to 2.0 config-translator Tool"
+short-title: "Using the 1.0 to 2.0 config-translator Tool"
+description: "Instructions for using the experimental Translator to translate CircleCI 1.0 configuration to the 2.0 format."
+categories: [migration]
+hide: true
+order: 60
+---
+
+This document provides instructions for using the `config-translator` tool to generate a basic working CircleCI 2.0 configuration from your existing CircleCI 1.0 project for a limited set of languages:
+* Ruby
+* PHP
+* Node.js
+* iOS (partial: 1.0 code signing is not supported - use Fastlane instead)
+* Java (partial)
+
+The generated `config-translation` file includes comments to help you understand how to modify the configuration for use as the `.circleci/config.yml` in your CircleCI 2.0 project. 
+
+**Warning: Do not use the generated config file in production without testing.** There are structures in 1.0 that cannot be reconciled in the 2.0 configuration, so it is important to put the generated configuration file on a branch, test it, and modify it before going to production. Results may vary significantly from your 1.0 results with the draft configuration generated by `config-translator`.
+
+## Overview
+The `config-translator` is a tool to help you migrate projects from CircleCI 1.0 to 2.0. Major differences exist between CircleCI 1.0 and 2.0 as follows:
```

Comment:
We are trying to avoid selling this as a "migration" solution -- consider instead "...is a tool to generate 2.0 configuration that shows both your 1.0 build steps expressed in the 2.0 syntax as well as explicitly call commands that in were inferred and run implicitly in your 1.0 builds."